backend-lucene/src/main/java/org/hibernate/search/v6poc/backend/lucene/types/predicate/impl/GeoPointSpatialWithinBoundingBoxPredicateBuilder.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.v6poc.backend.lucene.types.predicate.impl;
+
+import org.apache.lucene.document.LatLonPoint;
+import org.apache.lucene.search.Query;
+import org.hibernate.search.v6poc.backend.lucene.search.predicate.impl.AbstractSpatialWithinBoundingBoxPredicateBuilder;
+import org.hibernate.search.v6poc.backend.spatial.GeoPoint;
+
+class GeoPointSpatialWithinBoundingBoxPredicateBuilder extends AbstractSpatialWithinBoundingBoxPredicateBuilder<GeoPoint> {
+
+	GeoPointSpatialWithinBoundingBoxPredicateBuilder(String absoluteFieldPath) {
+		super( absoluteFieldPath );
+	}
+
+	@Override
+	protected Query buildQuery() {
+		return LatLonPoint.newBoxQuery( absoluteFieldPath, boundingBox.getBottomRight().getLatitude(), boundingBox.getTopLeft().getLatitude(),
+				boundingBox.getTopLeft().getLatitude(), boundingBox.getBottomRight().getLongitude() );
```

Comment:
I see *three* calls to `getLatitude` here, is it normal? Also, if the tests didn't catch that, you might want to add some tests.
If it's normal, maybe a comment would help here...

Reply:
I added a test.